packages/rds/initial-bootstrap.ts

```diff
@@ -0,0 +1,88 @@
+import { getAuthPassword } from "./auth/secrets";
+import { createConnection, Connection } from "typeorm";
+
+import { CloudFormationCustomResourceEvent, CloudFormationCustomResourceResponse } from "aws-lambda";
+
+type User = {
+  name: string;
+  dbPrivileges: string[];
+};
+
+const DB_NAME = "atat";
```

Comment:
Shall we go ahead and parameterize the DB name in anticipation of sharing a single RDS instance in Sandbox?

Reply:
Done -- it can be provided using the `DatabaseName` property on the `CustomResource`.

Reply:
If for some reason we can't set extra properties on the `CustomResource` or `Provider`, I'll just make it pull from an environment variable instead.